src/DynamoPackages/PackageLoader.cs

```diff
@@ -292,22 +310,25 @@ private void TryLoadPackageIntoLibrary(Package package)
                 PythonServices.PythonEngineManager.Instance.
                     LoadPythonEngine(package.LoadedAssemblies.Select(x => x.Assembly));
             }
-            catch (CustomNodePackageLoadException e)
+            catch (Exception e)
             {
-                Package originalPackage =
-                    localPackages.FirstOrDefault(x => x.CustomNodeDirectory == e.InstalledPath);
-                OnConflictingPackageLoaded(originalPackage, package);
+                // Try to load any valid node libraries even if the package is in error state
+                PackagesLoaded?.Invoke(loadedNodeLibs);
 
                 package.LoadState.SetAsError(e.Message);
-            }
-            catch (Exception e)
-            {
-                if (e is DynamoServices.AssemblyBlockedException)
+
+                if (e is CustomNodePackageLoadException ce)
+                {
+                    Package originalPackage =
+                        localPackages.FirstOrDefault(x => x.CustomNodeDirectory == ce.InstalledPath);
+                    OnConflictingPackageLoaded(originalPackage, package);
+                }
+                else if (e is DynamoServices.AssemblyBlockedException)
```

Comment:
shouldn't this be an `if`?

Reply:
well...it cannot be both exception types at once.
`else if` seems to enforce that fact a bit clearer

Reply:
not an issue anymore